kratos/utilities/global_pointer_utilities.h

```diff
@@ -27,6 +27,7 @@
 #include "containers/global_pointers_vector.h"
 #include "containers/global_pointers_unordered_map.h"
 #include "utilities/communication_coloring_utilities.h"
+#include "spatial_containers/spatial_search_result.h"
```

Comment:
i don't think you should add this include here...global_pointer_utilities are designed to be added anywhere in the code...

Reply:
Look why it is included

Reply:
I noticed that after your suggestions it is not needed anymore!